recipes/jinja2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version="2.8" %}
+
+package:
+  name: jinja2
+  version: {{ version }}
+
+source:
+  fn: Jinja2-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/J/Jinja2/Jinja2-{{ version }}.tar.gz
+  md5: edb51693fe22c53cee5403775c71a99e
+
+build:
+  number:  0
+  script: python -m pip install --no-deps .
```

Comment:
That's why I was thinking it should go in the example recipe. That way one can simply copy it from there. Alternatively, one can start with the example recipe as it was initially intended.